src/docs/product/integrations/integration-platform/index.mdx

```diff
@@ -275,105 +285,63 @@ For an example of how to build an internal integration, see [our Round Robin Iss
 
 #### We use the Sentry API internally at our company. Should we switch to the Sentry Integration Platform?
 
-If you're looking for any of the new features below, we recommend you switch to the Sentry Integration Platform utilizing the [Internal Integrations](#internal-integrations).
+If you're looking for any of the new features below, we recommend you switch to Sentry Integrations and build an [Internal Integration](#internal-integrations).
 
 **Permissions**
+OAuth apps and Auth Tokens allow you to access Sentry as a specific user in your Sentry organization. This means:
 
-OAuth apps and Auth Tokens allow you to access Sentry as a specific user in your Sentry org. This means:
-
-- If this user leaves the org or their permissions change, the integration can break. A Sentry Integrations App, on the other hand, is a separate, independent entity with its own permissions.
+- If this user leaves the organization or their permissions change, the integration can break. A Sentry Integration, on the other hand, is a separate, independent entity with its own permissions.
 - OAuth apps and Auth Tokens permit the user to access other Sentry organizations they belong to, so there is limited isolation.
 - Actions taken in Sentry will connect to this user rather than the integration/app. For example, if your alerting integration automatically assigns issues to a teammate, it will appear in the history as “Alice assigned Bob issue XYZ” rather than “&lt;Your Sentry Integration apps&gt; assigned Bob issue XYZ.”
 
 **Managing Webhooks**
-
-With Sentry Integration apps you can manage webhooks in the UI, as opposed to the current state where you need to make API calls to create/update/delete/retrieve webhooks. The latter is not only more cumbersome but also harder to keep track of and maintain visibility across your team.
+With Sentry Integrations you can manage webhooks in the UI, as opposed to the current state where you need to make API calls to create/update/delete/retrieve webhooks. The latter is not only more cumbersome but also harder to keep track of and maintain visibility across your team.
 
 **Scope**
-
 Currently, webhooks must be configured per-project, which is unnecessary overhead especially when adding/removing projects. With Sentry Integration apps, you’ll receive notifications for all projects in your organization.
 
-**New Event Types**
-
-Sentry Integration apps expose a richer set of events that are not available using the legacy service hooks API. Currently, this includes the following webhooks:
+**Event Types**
+Sentry Integrations expose a richer set of events that are not available using the legacy service hooks API. Currently, this includes the following webhooks:
 * Issue Created
 * Issue Resolved
 * Issue Ignored
 * Issue Assigned
 * Comment Created
 * Comment Updated
 * Comment Deleted
+* Error Created
```

Comment:
This is a comment for line 347 which Github won't let me select for some reason. Just change "Sentry Integrations" to "Sentry integrations."